tests/unit/amo/components/TestAddon.js

```diff
@@ -961,6 +961,24 @@ describe(__filename, () => {
     expect(root.find(AddonRecommendations)).toHaveLength(0);
   });
 
+  it('does not render recommendations if the add-on is not an extension', () => {
+    const fakeConfig = getFakeConfig({ enableAddonRecommendations: true });
+    for (const addonType of [
+      ADDON_TYPE_COMPLETE_THEME,
+      ADDON_TYPE_DICT,
+      ADDON_TYPE_LANG,
+      ADDON_TYPE_OPENSEARCH,
+      ADDON_TYPE_THEME,
+    ]) {
```

Comment:
I tend to write such loops in a slightly different manner to make the "for each" part more explicit (see below), but I guess I am fine with this way too!

```
[
  ADDON_TYPE_COMPLETE_THEME,
  ...
].forEach((addonType) => {
});
```

Reply:
Tbh, I prefer the `for... of` syntax over `forEach`, and, if it matters at all, it is apparently more performant. I've read that a number of places including [here](https://coderwall.com/p/kvzbpa/don-t-use-array-foreach-use-for-instead). So I would like to keep this code as is, which you say you're fine with too.